lib/round_vote.go

```diff
@@ -0,0 +1,127 @@
+package sebak
```

Comment:
Why isn't it in `round` package ?

Reply:
@Geod24 `sebak. Ballot` is used in `RoundVote`, and `round.Round` is used in sebak.Ballot.
Then it's going to be recursive import.
So if we want to modify to node, it's more complicated. We'll be able to handle it next time :)